### DISCUSSION TITLE
Welcome to openmessaging-storage-dledger Discussions!

### DISCUSSION THREAD

**Discussion by vongosling** [openmessaging/dledger#80] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pinocchio2018** [comment]
hi, What does the word Dledger mean?

**Reply by biningo** [reply]
Another name for dledger is d-ledger, which can be understood as ledger storage. @vongosling Do I understand this right?
